components/ResourceDetail/Masthead.vue

```diff
@@ -338,6 +338,7 @@ export default {
           <span v-else-if="isWorkspace">{{ t("resourceDetail.masthead.workspace") }}: <nuxt-link :to="workspaceLocation">{{ namespace }}</nuxt-link></span>
           <span v-else-if="namespace">{{ t("resourceDetail.masthead.namespace") }}: <nuxt-link :to="namespaceLocation">{{ namespace }}</nuxt-link></span>
           <span v-if="parent.showAge">{{ t("resourceDetail.masthead.age") }}: <LiveDate class="live-date" :value="get(value, 'metadata.creationTimestamp')" /></span>
+          <span v-if="value.showPodRestarts">{{ t("resourceDetail.masthead.restartCount") }}:<span class="live-date"> {{ value.restartCount }}</span></span>
```

Comment:
Very minor, but we might want to consider renaming the class `live-date` to something like `live-data`. It seems to style more than just a date.

Reply:
changed the class name to `live-data`